src/insert.rs

```diff
@@ -143,6 +109,12 @@ impl<T> Insert<T> {
         self
     }
 
+    /// Similar to [Client::with_option], but for this particular INSERT statement only.
+    pub fn with_option(mut self, name: impl Into<String>, value: impl Into<String>) -> Self {
```

Comment:
This method can be called after `write()`, which obviously not what user wanted. It's a programmer error (not an operational one), so we can panic in this case (and write about it in the doc comment).

Reply:
Should be resolved now.